controllers/ccruntime_controller.go

```diff
@@ -226,6 +226,20 @@ func (r *CcRuntimeReconciler) processCcRuntimeDeleteRequest() (ctrl.Result, erro
 			if r.ccRuntime.Spec.Config.PostUninstall.Image == "" {
 				controllerutil.RemoveFinalizer(r.ccRuntime, RuntimeConfigFinalizer)
 			} else if r.ccRuntime.Spec.Config.PostUninstall.Image != "" {
+				// FXIME: This should be treated in a better way, as just having the sleep
```

Comment:
nit: FXIME-> FIXME
This looks reasonable. We'll focus on a proper fix in the next release

Reply:
I will fix the fixme in the next iteration, just in order to save some trees and unblock peer-pods folks. :-)